package.json

```diff
@@ -11,7 +11,7 @@
     "client"
   ],
   "engines": {
-    "node": ">= 6.11.5"
+    "node": ">= 10.20.1"
```

Comment:
Return this to `10.13`

Reply:
I think 10.10 is better than 10.13 because this version supports http2 as a stable.

Reply:
10.13 is LTS and we support LTS

Reply:
Also webpack@5 support 10.13 as minimum, so we should support this range too

Reply:
> webpack@5 support 10.13 as minimum

ok, your answer is good for me. but 10.13 isn't minimum LTS.

Reply:
@hiroppy 

10.12.0 - is not LTS https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V10.md#2018-10-10-version-10120-current-targos

10.13.0 - is LTS https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V10.md#2018-11-27-version-10140-dubnium-lts-rvagg

So we should support minimum supported LTS and now it is 10.13

Reply:
thank you for sharing this info.